### PR TITLE
Prepare correct list of corePlugins

### PR DESCRIPTION
**Pull request by Fatme**
Should be merged after this https://github.com/telerik/mobile-cli-lib/pull/146

### CODE REVIEW

**Comment by rosen-vladimirov** [comment]
:+1:

File: lib/project/cordova-project.ts [hunk 1]
```diff
@@ -78,7 +78,8 @@ export class CordovaProject extends frameworkProjectBaseLib.FrameworkProjectBase
 	}
 
 	public adjustBuildProperties(buildProperties: any, projectData?: IProjectData): any {
-		buildProperties.CorePlugins = projectData.CorePlugins || [];
+		var configurationName = (options.release || options.r) ? "release" : "debug";
```
Comment (teobugslayer) [inline lib/project/cordova-project.ts]: Shouldn't this be just (options.release)?

**Comment by teobugslayer** [comment]
Also, the build states:

lib/project/cordova-project.ts(81,28): error TS2304: Cannot find name 'options'.
lib/project/cordova-project.ts(81,47): error TS2304: Cannot find name 'options'.
lib/project/cordova-project.ts(82,38): error TS2339: Property '$project' does not exist on type 'CordovaProject'.

File: lib/project/cordova-project.ts (hunk 1 above)
Comment (rosen-vladimirov) [inline lib/project/cordova-project.ts]: In fact yes, in master branch we can use only options.release now (if my fix is working as expected)

File: lib/project/framework-project-base.ts [hunk 2]
```diff
@@ -69,6 +71,25 @@ export class FrameworkProjectBase implements Project.IFrameworkProjectBase {
 		}
 	}
 
+	public getProperty(propertyName: string, configuration: string): any {
+		if(!this.projectInformation) {
+			this.$errors.fail("projectInformation object is empty");
+		}
```
Comment (teobugslayer) [inline lib/project/framework-project-base.ts]: This looks more like an "assert" to me. Can this condition be triggered by user actions?

Comment (Fatme) [inline lib/project/framework-project-base.ts]: Yep, good idea!

**Comment by Fatme** [comment]
run ci

File: lib/project/framework-project-base.ts [hunk 3]
```diff
@@ -69,6 +73,23 @@ export class FrameworkProjectBase implements Project.IFrameworkProjectBase {
 		}
 	}
 
+	public getProperty(propertyName: string, configuration: string): any {
+		assert.ok(this.projectInformation, "ProjectInformation object is null or undefined.");
+
+		var propertyValue: any = null;
+
+		if(this.projectInformation.hasBuildConfigurations) {
```
Comment (rosen-vladimirov) [inline lib/project/framework-project-base.ts]: why do we need hasBuildConfigurations? Can't we use directly:
```Typescript
var configData = this.projectInformation.configurationSpecificData[configuration];
if(configData) {
	propertyValue = configData[propertyName];
} else { 
        propertyValue = this.projectInformation.projectData[propertyName];
}
```

**Comment by rosen-vladimirov** [comment]
One minor comment, but :+1:  from me :)

**Comment by Fatme** [comment]
run ci

**Comment by Fatme** [comment]
run ci

**Pull request merged**